server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraObjectStoreModule.java

```diff
@@ -34,10 +34,10 @@
     @Override
     protected void configure() {
         bind(CassandraBlobsDAO.class).in(Scopes.SINGLETON);
-        bind(CassandraBlobId.Factory.class).in(Scopes.SINGLETON);
+        bind(HashBlobId.Factory.class).in(Scopes.SINGLETON);
```

Comment:
Then, _HashBolbId_ binding can be done in a new common module **ObjectStoreModule**.
BTW, this comment is not really important for now.